sections/gcloud.zsh

```diff
@@ -0,0 +1,40 @@
+#
+# Google Cloud Platform (gcloud)
+#
+# gcloud is a tool that provides the primary command-line interface to Google Cloud Platform. 
+# Link: https://cloud.google.com/sdk/gcloud/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GCLOUD_SHOW="${SPACESHIP_GCLOUD_SHOW=true}"
+SPACESHIP_GCLOUD_PREFIX="${SPACESHIP_GCLOUD_PREFIX="using "}"
+SPACESHIP_GCLOUD_SUFFIX="${SPACESHIP_GCLOUD_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_GCLOUD_SYMBOL="${SPACESHIP_GCLOUD_SYMBOL="☁️ "}"
```

Comment:
Please check that this icon doesn't have the same issue that the `aws` one.

Reply:
Fixed in https://github.com/denysdovhan/spaceship-prompt/pull/472/commits/a63a2fb910dd63ee6a70334f90932d5e1b272344